subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -1327,14 +1327,25 @@ static inline u32_t isr_rx_scan(u8_t devmatch_ok, u8_t devmatch_id,
 		       access_addr[0], &conn->access_addr[0], 4);
 		memcpy(&pdu_adv_tx->payload.connect_ind.lldata.crc_init[0],
 		       &conn->crc_init[0], 3);
-		pdu_adv_tx->payload.connect_ind.lldata. win_size = 1;
+		pdu_adv_tx->payload.connect_ind.lldata.win_size = 1;
 
 		conn_interval_us =
 			(u32_t)_radio.scanner.conn_interval * 1250;
 
 		conn_offset_us = radio_tmr_end_get() + 502 + 1250;
-		conn_offset_us -= radio_tx_chain_delay_get(0, 0);
-		conn_offset_us -= radio_tx_ready_delay_get(0, 0);
+
+		/* Compensation for negative remainder values,
+		 * refer to radio_tmr_start in hal/radio.c
+		 */
+		if (!_radio.remainder_anchor ||
+		    (_radio.remainder_anchor & BIT(31))) {
+			conn_offset_us -= TICKER_TICKS_TO_US(1);
+		}
+
+		/* Uncomment to check, with different window offsets,
```

Comment:
Seeing this, it seems that not only IP networking people would be interested in fault injection framework https://jira.zephyrproject.org/browse/ZEP-2118 , so would be nice that more people started talking to it ;-).